### PR TITLE
[stable9] Run the license script

### PR DESCRIPTION
**Pull request by nickvergessen**
Similar to #507

### CODE REVIEW

**Comment by MorrisJobke** [comment]
👍

File: apps/admin_audit/appinfo/app.php [hunk 1]
```diff
@@ -1,5 +1,8 @@
 <?php
 /**
+
```
Comment (rullzer) [inline apps/admin_audit/appinfo/app.php]: newline?

Comment (nickvergessen) [inline apps/admin_audit/appinfo/app.php]: script issue

**Comment by rullzer** [comment]
:+1:

**Pull request merged**